ush/ioda/bufr2ioda/bufr2ioda_adpupa_prepbufr.py

```diff
@@ -0,0 +1,443 @@
+# (C) Copyright 2023 NOAA/NWS/NCEP/EMC
```

Comment:
Found it necessary to add `#!/usr/bin/env python3 ` as first line in order to run this script from g-w `prepatmiodaobs`.

Reply:
Yes it it required to make python script an executable.